packages/ptokens-assets-evm/src/lib/index.ts

```diff
@@ -18,20 +39,8 @@ export function offChainFormat(_amount: BigNumber.Value, _decimals: number) {
   return BigNumber(_amount).dividedBy(BigNumber(10).pow(_decimals))
 }
 
-// export async function getAccount(_web3: Web3): Promise<string> {
-//   if (_web3.eth.defaultAccount) return _web3.eth.defaultAccount
-//   const accounts = await _web3.eth.getAccounts()
-//   return accounts[0]
-// }
-
-// export function getContract(_web3: Web3, _abi: Abi, _contractAddress: string, _account: string = undefined) {
-//   const contract = new _web3.eth.Contract(_abi, _contractAddress)
-//   contract.defaultAccount = _account
-//   return contract
-// }
-
-export async function getGasLimit(_publicClient: PublicClient) {
-  const block = await _publicClient.getBlock({blockTag: 'latest'})
+export async function getGasLimit(_publicClient: PublicClient): Promise<bigint> {
```

Comment:
Do we need this function? If yes, I would move it into pTokensEvmProvider, otherwise remove it

Reply:
removed